### DISCUSSION TITLE
Continuous Integration to my IOS project with TCA

### DISCUSSION THREAD

**Discussion by saroar** [pointfreeco/swift-composable-architecture#763] [Q&A]
HI after long hard work I have done one open-source project now I like to add CI what is a less cost or cost freeway I can run my CI here is my project https://github.com/AddaMeSPB/AddaMeIOSSwiftUI

o just start enjoying test with TCA

**Comment by lukeredpath** [comment]
Have you tried GitHub actions?

**Reply by saroar** [reply]
yes, I am trying to follow and understand @lukeredpath https://github.com/pointfreeco/isowords/tree/main/.github this
